data/json/bionics.json

```diff
@@ -859,7 +859,7 @@
     "id": "bio_scent_vision",
     "type": "bionic",
     "name": "Scent Vision",
-    "description": "While this system is powered, you're able to visually sense your own scent, making it possible for you to recognize your surroundings even if you can't see it.",
+    "description": "While this system is powered, you're able to visually sense your own scent, making it possible for you to recognize your surroundings even if you can't see them.",
```

Comment:
`it` referred to `your surroundings`, I suppose `them`refers to other creature but then you need to change `your surroundings` by something else.